.travis.yml

```diff
@@ -0,0 +1,44 @@
+language: python
+sudo: false
+env:
+  global:
+    - LD_PRELOAD=/lib/x86_64-linux-gnu/libSegFault.so
+    - SEGFAULT_SIGNALS=all
+
+matrix:
+  include:
+    - env: TOXENV=flake8
+    - env: TOXENV=3.4
+      python: 3.4
+    - env: TOXENV=3.5
+      python: 3.5
+    - env: TOXENV=3.6
+      python: 3.6
+
+before_install:
+  - python3 --version
+  - uname -a
+  - lsb_release -a
+  - virtualenv --version
+  - easy_install --version
+  - pip --version
+  - tox --version
```

Comment:
I was wrong. :grinning: This command can't be successful as it will be installed in the `install` section. Move this line to the install section.